internal/provider/models/team.go

```diff
@@ -64,3 +80,55 @@ func (data *Team) ReadFromResponse(ctx context.Context, team *iam.Team) diag.Dia
 
 	return nil
 }
+
+func (data *TeamResource) ReadFromResponse(ctx context.Context, team *iam.Team, memberIds *[]string) diag.Diagnostics {
+	var diags diag.Diagnostics
+	data.Id = types.StringValue(team.Id)
+	data.Name = types.StringValue(team.Name)
+	if team.Description != nil && *team.Description != "" {
+		data.Description = types.StringValue(*team.Description)
+	} else {
+		data.Description = types.StringNull()
+	}
+	if memberIds != nil && len(*memberIds) > 0 {
+		data.MemberIds, diags = utils.StringSet(memberIds)
+		if diags.HasError() {
+			return diags
+		}
+	} else {
+		data.MemberIds = types.SetNull(types.StringType)
+	}
```

Comment:
wrote it this way to prevent this error:

```
│ Error: Provider produced inconsistent result after apply
│ 
│ When applying changes to astro_team.example1, provider "provider[\"registry.terraform.io/astronomer/astro\"]" produced an unexpected new value: .member_ids: was null, but now cty.SetValEmpty(cty.String).
│ 
│ This is a bug in the provider, which should be reported in the provider's own issue tracker.

```